osgpkitools/ConnectAPI.py

```diff
@@ -59,14 +59,15 @@ def request_unauthenticated(self, config, name, email, phone, csr, vo='', cc_lis
         for conn in [httplib.HTTPSConnection(host_no_port), httplib.HTTPConnection(host_no_port)]:
```

Comment:
Need to add `last_network_error = None` right above this to avoid potential NameError below